src/main/java/hudson/plugins/testlink/AbstractTestLinkBuilder.java

```diff
@@ -361,4 +371,26 @@ public Action getProjectAction(AbstractProject<?, ?> project) {
         return customFieldNamesArray;
     }
 
+
+    protected String[] createArrayOfTestPlanCustomFieldsNames(final VariableResolver<String> variableResolver,
```

Comment:
Are you reading my mind? 
I was going to make it later after the functionality merge!
You are so thoughtful